.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report for MCprep! We just need some info
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+    - type: textarea
```

Comment:
Can we add one more question which is "Did the issue continue after you restarted blender?"
and they have to select Yes or no (but if they select no, then they really shouldn't continue filling out the form).

So many bug reports have been resolved due to this simple thing, often after someone did an update but didn't restart blender.